OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -314,14 +328,31 @@ public HackyAI(HackyAIInfo info, ActorInitializer init)
 					&& !unit.Info.HasTraitInfo<HuskInfo>()
 					&& unit.Info.HasTraitInfo<ITargetableInfo>();
 
-			unitCannotBeOrdered = a => a.Owner != Player || a.IsDead || !a.IsInWorld;
+			unitCannotBeOrdered = UnitCannotBeOrdered;
 
 			foreach (var decision in info.PowerDecisions)
 				powerDecisions.Add(decision.OrderName, decision);
 
 			maximumCaptureTargetOptions = Math.Max(1, Info.MaximumCaptureTargetOptions);
 		}
 
+		bool UnitCannotBeOrdered(Actor a)
+		{
+			if (a.Owner != Player || a.IsDead || !a.IsInWorld)
+				return true;
+
+			var tag = a.TraitOrDefault<ScriptTags>();
```

Comment:
I haven't had the time / ability to sit down and give this attention it needs to comment properly.  I will get to it when I can.